psql2mysql.py

```diff
@@ -236,10 +236,12 @@ def do_migration(config):
                           title="Commands",
                           help="Available commands",
                           handler=add_subcommands),
-        cfg.StrOpt('source',
+        cfg.URIOpt('source',
+                   schemes = [ "postgresql", "mysql", "mysql+pymsql"],
                    required=True,
                    help='connection URL to the src DB server'),
-        cfg.StrOpt('target',
+        cfg.URIOpt('target',
+                   schemes = [ "postgresql", "mysql", "mysql+pymsql"],
```

Comment:
I think we want to support only "mysql+pymsql", right?

Reply:
Agreed. Some queries on the target side are MySQL/MariaDB specific. And we're only testing with mysq+pymysql currently